src/components/registration/registration.jsx

```diff
@@ -0,0 +1,45 @@
+import React,{useState} from 'react'
+import './registration.css'
+import logo from './school.svg'
+
+export default class Registration extends React.Component{
+
+    state={
+        registrationForm: 'display-none',
+        loginForm: ''
+    }
+
+
+    render(){
+        return(
+            <React.Fragment>
+               <div className='login-page' >
+                   <div className='form'>
+                       <img src={logo} alt="logo" className='logo'/>
+                       <form className={this.state.registrationForm}>
+                       <input type="text" placeholder='name'/>
+                       <input type="password" placeholder='password'/>
+                       <input type="text" placeholder='email address'/>
```

Comment:
I think 'address' in 'email address' is redundant. It would be better just 'email'